src/main/kotlin/uk/gov/dluhc/printapi/config/SchedulerLockConfiguration.kt

```diff
@@ -11,7 +12,7 @@ import software.amazon.awssdk.services.dynamodb.DynamoDbClient
 @Configuration
 @EnableScheduling
 @ConditionalOnProperty("jobs.enabled", havingValue = "true")
-// @EnableSchedulerLock(defaultLockAtMostFor = "PT5M")
+@EnableSchedulerLock(defaultLockAtMostFor = "\${jobs.lock-at-most-for}")
 class SchedulerLockConfiguration(
```

Comment:
I've moved the `lock-at-most-for` to a level up which will be globally set for all jobs

Reply:
I think @fayimora has also done this (based on your comment on his PR)
His PR is ready to be merged and he will merge it shortly. His PR needs to be merged before yours

Reply:
@NathanRussellValtech Thanks for updating me. This PR is now rebased from main (changes pushed by @fayimora) and is ready for review